_posts/2017-10-25-geopandas-tutorial.md

```diff
@@ -69,7 +69,7 @@ def recursive_query(target_area=None, full_url=None):
     
     if target_area:
         bbox = ','.join(map(lambda x: str(x), target_area.bounds))
-        resp = requests.get(url='https://transit.land/api/v1/operators', params={'bbox': bbox, 'per_page': 50})
+        resp = requests.get(url='https://Transitland/api/v1/operators', params={'bbox': bbox, 'per_page': 50})
```

Comment:
is there some contextual text parsing during the build process that will convert that to transit.land - or a reason why it was switched from transit.land to Transitland?

Reply:
Just some excessive use of multi-cursor text editing.